Dockerfile

```
@@ -1,28 +1,38 @@
-FROM node:10-alpine
+FROM node:10-alpine as base
 LABEL Description="Web Frontend of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
 
 # Expose the app port
-EXPOSE 3000
-
+ARG BUILD_COMMIT
```

You removed the port!

```suggestion
EXPOSE 3000
ARG BUILD_COMMIT
```

---

we didn't remove the port, it is here https://github.com/Human-Connection/Nitro-Web/blob/9f8a292c7e39030a4fb05622c8830e5dbff967db/Dockerfile#L27... the `CMD` and `EXPOSE`  do not need to be in a particular line in the Dockerfile, they are not used during the build, they are only used 1) during deploy 2) when using `docker-compose up`

---

Ahh allright I just saw the "removal" in the diff but missed that its now on the end of the file.

---

yeah, no worries... I think that we could definitely do a refactor once we get it reliably passing the build... maybe I should put a [WIP] tag on the `PR`...